stdlib/public/Concurrency/Task.swift

```diff
@@ -77,6 +77,38 @@ import Swift
 /// This reflects the fact that a task can be canceled for many reasons,
 /// and additional reasons can accrue during the cancellation process.
 ///
+/// A `Task` becomes cancelled when its `cancel()` method is called on it. There
+/// is no other mechanism by which a `Task` can become cancelled. This is true
```

Comment:
Thats very sneaky wording I’m not comfortable with… its true for specifically an unstructured task but the wording here isn’t clear enough, it may sound like you mean “any task”